packages/expo-image/ios/ImageModule.swift

```diff
@@ -65,6 +65,14 @@ public final class ImageModule: Module {
         view.cachePolicy = cachePolicy ?? .disk
       }
 
+      Prop("accessible") { (view, accessible: Bool) in
+        view.accessible = accessible
+      }
+
+      Prop("accessibilityLabel") { (view, label: String) in
+        view.accessibilityLabel = label
```

Comment:
What about this? Then you don't need the `didSet` and separate member in `ImageView` 😉 
```suggestion
        view.sdImageView.accessibilityLabel = label
```

Reply:
😄 I was going to just do that but I thought you mightn't want to access it directly like that